xml/System/Version.xml

```diff
@@ -1584,10 +1595,41 @@ major.minor[.build[.revision]]
         <Parameter Name="version" Type="System.Object" Index="0" FrameworkAlternate="dotnet-uwp-10.0;netcore-1.0;netcore-1.1;netstandard-1.0;netstandard-1.1;netstandard-1.2;netstandard-1.3;netstandard-1.4;netstandard-1.5;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="version">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="version">An object to compare, or <see langword="null" />.</param>
+        <summary>Compares the current <see cref="T:System.Version" /> object to a specified object and returns an indication of their relative values.</summary>
+        <returns>A signed integer that indicates the relative values of the two objects, as shown in the following table:
+          <list type="table">
+            <listheader>
+              <term>Return value</term>
+              <description>Meaning</description>
+            </listheader>
+            <item>
+              <term>Less than zero</term>
+              <description>The current <see cref="T:System.Version" /> object is a version before <paramref name="version" />.</description>
+            </item>
+            <item>
+              <term>Zero</term>
+              <description>The current <see cref="T:System.Version" /> object is the same version as <paramref name="version" />.</description>
+            </item>
+            <item>
+              <term>Greater than zero</term>
+              <description>The current <see cref="T:System.Version" /> object is a version subsequent to <paramref name="version" />.
+
+-or-
```

Comment:
need to double check my change here will change the broken table
https://review.docs.microsoft.com/en-us/dotnet/api/system.version.system-icomparable-compareto?view=netstandard-1.6&branch=pr-en-us-3232&viewFallbackFrom=netcore-3.0

![image](https://user-images.githubusercontent.com/12971179/65559303-0ab9a780-deef-11e9-97be-64cfdc8d1398.png)

Reply:
This was fixed.